app/views/reservations/_form.html.erb

```diff
@@ -41,9 +41,6 @@
     </div>
   </div>
 
-  <%= label_tag "Add new notes" %>
-  <%= text_area_tag :new_notes, "", rows: 5, class: 'span4' %>
-  <br />
   <%= render :partial => 'shared/markdown_button' %>
```

Comment:
You should take this out of the form partial and put it inside the notes form on the show view (for both reservations and equipment items.